pemutil/pem.go

```diff
@@ -277,6 +278,52 @@ func ParseCertificateRequest(pemData []byte) (*x509.CertificateRequest, error) {
 	return nil, errors.New("error parsing certificate request: no certificate found")
 }
 
+// PEMType represents a PEM block type. (e.g., CERTIFICATE, CERTIFICATE REQUEST, etc.)
+type PEMType int
+
+func (pt PEMType) String() string {
+	switch pt {
+	case PEMTypeCertificate:
+		return "Certificate"
+	case PEMTypeCertificateRequest:
+		return "Certificate Request"
+	default:
+		return "Undefined"
+	}
+}
```

Comment:
You're using this to rerturn an error, but it would be quite weird to use upper cases there, e.g, `file mariano.crt does not contain a valid PEM encoded Certificate"`